lib/std/crypto/ripemd160.zig

```diff
@@ -0,0 +1,218 @@
+// https://homes.esat.kuleuven.be/~bosselae/ripemd160.html
+
+const std = @import("std");
+
+pub const Ripemd160 = struct {
+    bytes: [160 / 8]u8,
+
+    pub fn hash(str: []const u8) Ripemd160 {
```

Comment:
Input is not assumed to be a string, so `str` may not be a good name for that parameter :)